docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md

```diff
@@ -1 +1,150 @@
-# Working with Salesforce client
+Template for Working with Salesforce client
+
+# Working with Salesforce client 
+
+This template refers to the tutorial described in [Working with Salesforce client](https://ei.docs.wso2.com/en/latest/ballerina-integrator/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/1/). Please refer it for more details on what you are going to build here and the prerequisites for building the module .
+
+#### Executing the Template
+
+- Run the following command to pull the ```working_with_salesforce_client``` template from the Ballerina Central.
+
+```
```

Comment:
Shall we tab all code blocks so that they are in line with the bullet point?

Reply:
Fixed